application/shared-kernel/ApiCore/SinglePageApp/SinglePageAppConfiguration.cs

```diff
@@ -18,6 +18,7 @@ public class SinglePageAppConfiguration
     public static readonly string BuildRootPath = GetWebAppDistRoot("WebApp", "dist");
 
     private readonly string _htmlTemplatePath;
+    private readonly bool _isDevelopment;
```

Comment:
What problem is this solving. AFAIK this is no longer a problem. If this is fixing something, it means that we have not solve the root cource.

I suggest we revert all changes to this file, and address this in a separate pull request.